packages/app-mobile/components/screens/Note.tsx

```diff
@@ -612,11 +599,27 @@ class NoteScreenComponent extends BaseScreenComponent {
 			} catch (error) {
 				reg.logger().warn('Error when unlinking cached file: ', error);
 			}
-		} else {
-			await shim.fsDriver().copy(localFilePath, targetPath);
+			return true;
+		};
+
+		const canResize = dimensions.width > maxSize || dimensions.height > maxSize;
+		if (canResize) {
+			const resizeLargeImages = Setting.value('imageResizing');
+			if (resizeLargeImages === 'alwaysAsk') {
+				const userAnswer = await dialogs.pop(this, _('You are about to attach a large image (%dx%d pixels). Would you like to resize it down to %d pixels before attaching it? (You may change this default behaviour in the options)', dimensions.width, dimensions.height, maxSize), [
```

Comment:
Actually could you change the message to this please?

```js
_('You are about to attach a large image (%dx%d pixels). Would you like to resize it down to %d pixels before attaching it?') + '\n\n' + _('(You may change this disable this prompt in the options)')
```

So it should two translatable strings separated by newlines. Same for the other message